src/styles/StyleUtils.js

```diff
@@ -45,17 +79,46 @@ function getAvatarStyle(size) {
     };
 }
 
+/**
+* Return the border style if avatar is SVG
+*
+* @param {String} size
+* @param {String} type
+* @returns {Object}
+*/
+function getAvatarBorderRadius(size, type) {
+    if (type === CONST.ICON_TYPE_WORKSPACE) {
+        return {borderRadius: avatarBorderSizes[size]};
+    }
+
+    // Default to rounded border
+    return {borderRadius: variables.buttonBorderRadius};
+}
+
 /**
  * Return the border style if avatar is SVG
  *
- * @param {Boolean} isSVG
+ * @param {String} size
+ * @param {String} type
  * @returns {Object}
  */
-function getAvatarSVGBorder(isSVG) {
-    if (!isSVG) {
-        return {};
-    }
-    return styles.svgAvatarBorder;
+function getAvatarBorderStyle(size, type) {
+    return {
+        overflow: 'hidden',
+        ...getAvatarBorderRadius(size, type),
+    };
+}
+
+/**
+ * Helper method to return old dot default avatar associated with login
```

Comment:
Does Old dot use some similar color system?

Reply:
Old Dot has 7 default avatars that are also randomly selected from. Right now, Old Dot uses the stored avatar to get the default avatar. So when we delete a custom uploaded avatar, we need to replace it with the old version of avatars for it to work on Old Dot too

Reply:
Thanks for explaining